src/tools/nvmf-rbd-gw/test-nvmf-rbd-gw.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Nit: try to avoid mixing tests w/ the code. I'd recommend moving to a subdirectory under `src/test/`

Reply:
Ok.